substra/sdk/client.py

```diff
@@ -376,12 +376,20 @@ def add_traintuple(self, data, exist_ok=False):
 
         If `exist_ok` is true, `AlreadyExists` exceptions will be ignored and the
         existing asset will be returned.
+
+        If `fake_data` is true and the backend is local, the traintuple runs on fake data.
+        If `fake_data` is true and `n_fake_samples` is set, the traintuple
+        runs on n samples of fake data.
         """
         spec = self._get_spec(schemas.TraintupleSpec, data)
-        return self._backend.add(spec, exist_ok=exist_ok)
+        spec_options = {
+            'fake_data': fake_data,
+            'n_fake_samples': n_fake_samples
+        }
+        return self._backend.add(spec, exist_ok=exist_ok, spec_options=spec_options)
 
     @logit
-    def add_aggregatetuple(self, data, exist_ok=False):
+    def add_aggregatetuple(self, data, exist_ok=False, fake_data=False, n_fake_samples=None):
```

Comment:
There are no data samples for aggregate tasks, I think you can remove this part.